chore: update readme (#355)
ExchangeUnion/xud-docker: README.md

### Developing a feature

Create a feature branch.

```
git checkout -b your-feature-branch
```

Make your desired changes to the images located at: `$PROJECT_DIR/images`.

Build your modified images. Optionally modify the `nodes.json` in the project root if necessary.

```bash
tools/build
```

Test it locally.

```bash
./xud.sh -b your-feature-branch
```

or

```bash
./xud.sh -b your-feature-branch --nodes-json ./nodes.json
```

To let others test without building the images by themselves push your feature branch to remote repository. Travis will build & push images for you.

```bash
git push origin your-feature-branch
```

> After corresponding Travis build turned green ask other people to run your feature branch on their machine.

Reviewer: ```suggestion
After the corresponding Travis build succeeded other people can easily run your feature branch on their machine like this.
```
